afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt

```diff
@@ -161,13 +161,15 @@ class AfterpayPriceBreakdown @JvmOverloads constructor(
                         AfterpayInfoSpan(infoUrl, false),
                         Spannable.SPAN_INCLUSIVE_EXCLUSIVE
                     )
+                    append(" ")
                 } else if (linkStyle.text != null) {
                     append(" ")
                     append(
                         resources.getString(linkStyle.text),
                         AfterpayInfoSpan(infoUrl, linkStyle.underlined),
                         Spannable.SPAN_INCLUSIVE_EXCLUSIVE
                     )
+                    append("\u200B")
```

Comment:
Should we have a brief comment here and on line 215 to explain what the zero-width space is for?

Reply:
This has now been removed as it provides a better experience.